npm-install/action.yaml

```diff
@@ -1,12 +1,17 @@
 name: Set up deps
 description: Loads NPM dependencies for a CI job, installing them if not cached
+inputs:
+  node-version:
+    description: Node version
+    required: false
+    default: '14'
```

Comment:
Can you make this required and not set a default policy? That will force anyone using this action to specify a desired Node.js version.